3-networks/README.md

```diff
@@ -16,114 +65,197 @@ The purpose of this step is to:
+### Deploying with Jenkins
 
-### Setup to run via Jenkins
-
-1. Clone the repo you created manually in bootstrap: `git clone <YOUR_NEW_REPO-3-networks>`.
-1. Navigate into the repo `cd YOUR_NEW_REPO_CLONE-3-networks` and change to a non production branch `git checkout -b plan` (the branch `plan` is not a special one. Any branch which name is different from `development`, `non-production` or `production` will trigger a terraform plan).
-1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/3-networks/ .` (modify accordingly based on your current directory).
-1. Copy the Jenkinsfile script `cp ../terraform-example-foundation/build/Jenkinsfile .` to the root of your new repository (modify accordingly based on your current directory).
+1. Clone the repo you created manually in 0-bootstrap.
+   ```
+   git clone <YOUR_NEW_REPO-3-networks>
+   ```
+1. Navigate into the repo and change to a non-production branch.
+   ```
+   cd YOUR_NEW_REPO_CLONE-3-networks
+   git checkout -b plan
+   ```
+1. Copy contents of foundation to new repo.
+   ```
+   cp -RT ../terraform-example-foundation/3-networks/ .
+   ```
+1. Copy the Jenkinsfile script to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/Jenkinsfile .
+   ```
 1. Update the variables located in the `environment {}` section of the `Jenkinsfile` with values from your environment:
     ```
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     _PROJECT_ID (the cicd project id)
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository (modify accordingly based on your current directory).
-1. Ensure wrapper script can be executed `chmod 755 ./tf-wrapper.sh`.
+1. Copy terraform wrapper script to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/tf-wrapper.sh .
+   ```
+1. Ensure wrapper script can be executed.
+   ```
+   chmod 755 ./tf-wrapper.sh
+   ```
 1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap. See any of the envs folder [README.md](./envs/production/README.md) files for additional information on the values in the `common.auto.tfvars` file.
 1. Rename `shared.auto.example.tfvars` to `shared.auto.tfvars` and update the file with the `target_name_server_addresses`.
 1. Rename `access_context.auto.example.tfvars` to `access_context.auto.tfvars` and update the file with the `access_context_manager_policy_id`.
-1. Commit changes with `git add .` and `git commit -m 'Your message'`.
-1. You will need to manually plan + apply the `shared` environment (only once) since development, non-production and production depend on it.
+1. Commit changes.
+   ```
+   git add .
+   git commit -m 'Your message'
+   ```
+1. You must manually plan and apply the `shared` environment (only once) since development, non-production and production depend on it.
```

Comment:
```suggestion
1. You must manually plan and apply the `shared` environment (only once) since the `development`, `non-production` and `production` environments depend on it.
```